recipes/custodian/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "custodian" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "30bb40a3eaa7dbbbfe01de74434d38ad4cdf4b5898b123b00631e9dcb194b5fa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
```

Comment:
Once you add the entry points, you will want to test each of them:

https://conda.io/docs/building/meta-yaml.html#test-commands